src/Symfony/Component/Security/Csrf/CsrfTokenManager.php

```diff
@@ -134,6 +134,9 @@ private function derandomize(string $value): string
             return $value;
         }
         $key = base64_decode(strtr($parts[1], '-_', '+/'));
+        if (0 === \strlen($key)) {
```

Comment:
```suggestion
        if ('' === $key || false === $key) {
```

To make fabbot happy.